org.eclipse.scanning.test/src/org/eclipse/scanning/test/command/PyExpresserTest.java

```diff
@@ -310,14 +337,14 @@ public void testScanRequestWithProcessing() throws Exception {
 		gmodel.setSnake(false);
 
 		ScanRequest<IROI> request = new ScanRequest<>();
-		request.setCompoundModel(new CompoundModel(Arrays.asList(gmodel)));
+		request.setCompoundModel(new CompoundModel<IROI>(Arrays.asList(gmodel)));
 		Map<String,Object> detectors = new LinkedHashMap<>();
 		detectors.put("mandelbrot", new MandelbrotModel("p", "q"));
 		detectors.put("processing", new ClusterProcessingModel("processing", "mandelbrot", "/tmp/something.nxs"));
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "processing" 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)